### PR TITLE
Added timeframe start/end for poll objects

### LINKED ISSUE #6: As an organizer, I can set a custom timeframe using ISO timestamps
- [ ] create TimeInterval model
- [x] have poll creation page take in two textboxes for iso start and iso stop
- [x] add validation (stop > start)
- [x] update models for poll to store start/stop

### LINKED ISSUE #7: As an organizer, I can see the timeframe reflected on the custom link
- [ ] query db to display start/stop

### PR DESCRIPTION
**Pull request by erinpwoo**
- Added DB migration to add SQL DateTime type for `timeframe_start` and `timeframe_end`.
- Modified views with start/end time input fields and their displays
- Checks valid start/end date time format before entry validation
- Validate form fields
- New tests for valid and invalid time inputs, modified polls fixture with new timeframe fields

Note: timeframe inputs should be in ISO 8601 format.

### CODE REVIEW

File: test/controllers/polls_controller_test.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 class PollsControllerTest < ActionDispatch::IntegrationTest
   setup do
-    @poll = polls(:one)
+    @poll = polls(:poll)
```
Comment (grakshith) [inline test/controllers/polls_controller_test.rb:5]: Is there a reason why one was changed to poll?

File: test/fixtures/polls.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 
-one:
+poll:
```
Comment (grakshith) [inline test/fixtures/polls.yml:3]: Same question as before.

**Review by grakshith — COMMENTED** [review]
LGTM pending the two comments.

File: test/controllers/polls_controller_test.rb (hunk 1 above)
Comment (akshayheda) [reply test/controllers/polls_controller_test.rb:5]: it was unclear what :one meant (not knowing it was a poll name). It seemed like it was pulling the first poll from some unknown list. This was also a side effect of us considering adding new fixtures, but deciding it was unneeded. The idea is in the future, we might add fixtures like "invalid_poll" or "invalid_title_poll".

Comment (grakshith) [reply test/controllers/polls_controller_test.rb:5]: Sounds good.

**Review by grakshith — APPROVED** [review]

**Pull request merged**